### ISSUE TITLE
Process reports empty listening address

### ISSUE THREAD

**Issue by alexaltair** [mhart/kinesalite#37]
When I run kinesalite, I get the following 
```
17:19:47 kinesalite.1 | started with pid 14538
17:19:48 kinesalite.1 | Listening at http://:::10500
```
The port is right, and it running on localhost. (There doesn't seem to be a functional problem, just an aesthetic one.)

**Comment by mhart** [comment]
That's correct! Welcome to IPv6 😊

http://serverfault.com/questions/444554/what-does-mean-as-an-ip-address-bracket-colon-colon-bracket

**Comment by alexaltair** [comment]
Ah, I see. That makes sense. Why then, does it use my IPv6 localhost address, rather than IPv4?

**Comment by mhart** [comment]
This has just been a Node.js default for a while now:

> If the hostname is omitted, the server will accept connections on any IPv6 address (::) when IPv6 is available, or any IPv4 address (0.0.0.0) otherwise

https://nodejs.org/api/net.html#net_server_listen_port_hostname_backlog_callback
